### ISSUE TITLE
new access

### ISSUE THREAD

**Issue by Geetus** [github/training-kit#464]
## Overview
**TL;DR**
Watched the video at the following link but found it confusing to follow as the issue it referes to is no longer valid. https://services.github.com/on-demand/intro-to-github/

Also: can i get on as a collaborator, many thanks

#### Screenshots
<If bug fix, please upload images of the error to better showcase the problem.>

#### Reasoning
<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by Geetus** [comment]
now on!

**Comment by hectorsector** [comment]
Thanks for the update, @Geetus! Could you please help us find a link of the vide with the outdated instructions? The only one I see is [here](https://services.github.com/on-demand/intro-to-github/02/) which looks up to date.

**Comment by Geetus** [comment]
@hectorsector 
Hi Hector. Thanks for the link.

Here is the link that I was trying to use. Following the instructions it was not possible to find my own handle using 'branch-master'..
[https://services.github.com/on-demand/intro-to-github/](url)
https://services.github.com/on-demand/intro-to-github/

**Comment by hectorsector** [comment]
@Geetus now that [you're a collaborator](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/issues/3653) you should be able to create your own branch [right on the open-enrollment repo](https://github.com/githubschool/open-enrollment-classes-introduction-to-github).

**Comment by brianamarie** [comment]
Hi @Geetus! You're right that the video is out of date, thank you for bringing that to our attention. CC/ @github/services-training for awareness: The instructions in the video tell users to comment on an issue, and that process has changed. @Geetus isn't the only one who has brought this up as a confusing issue.

**Comment by r5atom** [comment]
On [page 2](https://services.github.com/on-demand/intro-to-github/02/) _>"Tell me why">"Why Did We Ask You to Comment on this Issue?"_ the explanation assumes the new collaboration is granted by leaving a _**comment**_ rather than _**creating an issue**_.

**Comment by hectorsector** [comment]
You're absolutely right @r5atom! I can go ahead and correct it, but please feel free to try submitting a PR yourself if you get to it before I do!

**Comment by brianamarie** [comment]
Actually @r5atom and @hectorsector I've already started a PR [here](https://github.com/github/training-kit/pull/468), I'd love your reviews!

**Comment by hectorsector** [comment]
Looks like @hollenberry is going to update the video as referenced in #479 so I will close this one out as a duplicate. Thanks for reporting this @Geetus
